packages/api/smart.go

```diff
@@ -78,8 +78,8 @@ func validateSmartContract(r *http.Request, data *apiData, result *prepareResult
 					pref := getPrefix(data)
 					signature := &model.Signature{}
 					signature.SetTablePrefix(pref)
-					found, err := signature.Get(ret[1])
-					if err != nil {
+					var found bool
```

Comment:
what is the reason for explicit variable declaration?

Reply:
`err` shadowing